src/main/kotlin/net/axay/hglaborlobby/functionality/RandomFireworkCommand.kt

```diff
@@ -0,0 +1,34 @@
+package net.axay.hglaborlobby.functionality
+
+import net.axay.kspigot.utils.addEffect
+import net.axay.kspigot.utils.fireworkItemStack
+import org.bukkit.Color
+import org.bukkit.FireworkEffect
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import org.bukkit.inventory.ItemStack
+import kotlin.random.Random
+
+
+object RandomFireworkCommand : CommandExecutor {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+
+        val player: Player = sender as Player
+        if (!player.hasPermission("hglabor.lobby.randomfirework")) return false
```

Comment:
the prefix is just `lobby.`

Reply:
and return true not false

Reply:
was ist da bei true und false der unterschied